service/src/test/java/org/cbioportal/service/util/AlterationEnrichmentUtilTest.java

```diff
@@ -88,15 +89,15 @@ public void createAlterationEnrichments() throws Exception {
         AlterationEnrichment alterationEnrichment1 = result.get(0);
         Assert.assertEquals((Integer) 2, alterationEnrichment1.getEntrezGeneId());
         Assert.assertEquals("HUGO2", alterationEnrichment1.getHugoGeneSymbol());
-        Assert.assertEquals("CYTOBAND2", alterationEnrichment1.getCytoband());
+        Assert.assertEquals(null, alterationEnrichment1.getCytoband());
```

Comment:
if these are just null then let's remove? if there are plans to get the cytoband from the `reference_genome_gene` then use that instead.

Reply:
decided to use '-' if no cytoband found in reference_genome_gene table.

Reply:
we do not support multiple genome at the profile level, genetic alterations do not carry any information on the reference genome used.  But, that is okay since we can make a API call to query reference genome genes of queried genes from the frontend to retrieve their cytobands. So, in backend the cytoband of any alterations has a null value.